### PR TITLE
Update batching docs for Spanner and BigQuery

### PR DESCRIPTION
**Pull request by DoodleBobBuffPants**

### CODE REVIEW

File: documentation/database/big-query.md [hunk 1]
```diff
@@ -61,15 +61,20 @@ Support Level determines the degree of support available for this database ([lea
 
 ### Performance
 
-GCP BigQuery can suffer from performance issues while executing schema changes.
+Executing multiple schema changes against GCP BigQuery is comparatively slow as it is optimized for data. You can read more about it [here](https://cloud.google.com/blog/products/data-analytics/troubleshoot-bigquery-performance-with-these-dashboards).
 
-Flyway Teams Edition solves this via [batching](/documentation/configuration/parameters/batch) which combines schema changes to reduce the network overhead and improves performance.
+Flyway Teams Edition alleviates this via batching which executes multiple schema changes in one batch to minimize latency and improve performance.
```
Comment (Barry-RG) [inline documentation/database/big-query.md]: Don't like the term 'batching' being defined by the term 'batch'. Seems a little redundant. Could this be better said as 'via batching which executes multiple schema changes in one **request**'?

File: documentation/database/cloud-spanner.md [hunk 2]
```diff
@@ -59,11 +59,16 @@ Support Level determines the degree of support available for this database ([lea
 
 ## Flyway Teams Features for Cloud Spanner
 
-GCP Spanner can suffer from performance issues while executing schema changes.
+Executing multiple schema changes against Cloud Spanner is comparatively slow due to its need to validate your data. You can read more about it [here](https://cloud.google.com/spanner/docs/schema-updates#performance).
 
-Flyway Teams edition solves this via [batching](/documentation/configuration/parameters/batch) which combines schema changes to reduce the network overhead and improves performance.
+Flyway Teams Edition alleviates this via batching which executes multiple schema changes in one batch to minimize latency and improve performance.
```
Comment (Barry-RG) [inline documentation/database/cloud-spanner.md]: Same as above

**Review by Barry-RG — CHANGES_REQUESTED** [review]

File: documentation/database/big-query.md (hunk 1 above)
Comment (DoodleBobBuffPants) [reply documentation/database/big-query.md]: see latest commit

File: documentation/database/cloud-spanner.md (hunk 2 above)
Comment (DoodleBobBuffPants) [reply documentation/database/cloud-spanner.md]: see latest commit

**Review by Barry-RG — APPROVED** [review]

**Pull request merged**